popup/pages/AllPage.tsx

```diff
@@ -8,8 +9,32 @@ export const AllPage = () => {
   const search = useAtomValue(searchAtom);
   const entryIdToTags = useAtomValue(entryIdToTagsAtom);
 
+  const renderNoEntriesOverlay = () => {
```

Comment:
nit: Since it seems pretty similar I think we should just make this a component. Maybe something of the form like:

```
<NoEntriesOverlay title={string} subtitle={string} />
```

This is inspired by MUI's `DataGrid` slot by the way: https://mui.com/x/api/data-grid/data-grid/#DataGrid-css-noRowsOverlay